docs/PC002.md

```diff
@@ -2,26 +2,26 @@
 
 ## Cause
 
-You're calling an API in .NET Standard that isn't available in .NET Framework
+You're calling an API in .NET Standard that isn't available in the .NET Framework
 4.6.1.
 
-## Rule Description
+## Rule description
 
-While .NET Framework 4.6.1 is treated by NuGet as implementing .NET Standard
-2.0, it doesn't fully implement it. This was trade-off that is explained in the
-documentation of [.NET Standard][netfx-netstandard].
+While the .NET Framework 4.6.1 is treated by NuGet as implementing .NET Standard
+2.0, it doesn't fully implement it. This was a trade-off made that is explained in the
```

Comment:
> "This was a trade-off made that is explained in the"

You sure about the addition of "made"? Reads funny to me.

Reply:
I feel like it needs something there. @rpetrusha, what do you think? 
Perhaps it would read better as "This was a trade-off made, which is explained..."